src/browser/modules/D3Visualization/components/Explorer.tsx

```diff
@@ -166,42 +165,22 @@ export class ExplorerComponent extends Component<
     this.setHoveredItem(item)
   }
 
-  setHoveredItem = debounce(
-    (hoveredItem: VizItem) => this.setState({ hoveredItem }),
-    200
-  )
+  mounted = true
+  setHoveredItem = debounce((hoveredItem: VizItem) => {
+    if (this.mounted) {
+      this.setState({ hoveredItem })
+    }
+  }, 200)
 
   onItemSelect(selectedItem: VizItem): void {
     this.setState({ selectedItem })
   }
 
-  onGraphModelChange(stats: any) {
-    this.setState({ stats: stats })
+  onGraphModelChange(stats: GraphStats) {
+    this.setState({ stats })
     this.props.updateStyle(this.state.graphStyle.toSheet())
   }
 
```

Comment:
These calls/state turned out to be unneeded when I created the StyleableLabel/StyleableRelType components